action.yml

```diff
@@ -135,11 +140,22 @@ runs:
 
     - name: Download Product Zip Artifact
       uses: actions/download-artifact@v3
+      #if: ${{ inputs.do_zip_extract_step == true }}
+      if: ${{ inputs.do_zip_extract_step  == 'true'  && !contains(inputs.zip_name, '\n') }}
+      with:
+        path: ${{ env.ZIP_LOCATION }}
+        name: ${{ env.ZIP_NAME }}
+
+    - name: Download Product Zip Artifact
+      uses: actions/download-artifact@v3
+      #if: ${{ inputs.do_zip_extract_step == true }}
+      if: ${{ inputs.do_zip_extract_step  == 'true' && contains(inputs.zip_name, '\n') }}
       with:
         name: ${{ env.ZIP_NAME }}
         path: ${{ env.ZIP_LOCATION }}
```

Comment:
Is this a duplicate download?

Reply:
Okay, this was a mistake introduced by my revert of our first attempt at this in PR #41. Basically, I was trying to support the multi-archive code we added. I ended up dropping it because it didn't seem necessary anymore. This was leftover from trying to support it.